doc/2/guides/Asset Category/index.md

```diff
@@ -0,0 +1,74 @@
+---
```

Comment:
The directory should be named `asset-category`

Reply:
done